### PR TITLE
Fix sharing context between nodes

### PR DESCRIPTION
**Pull request by feodor-ra**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,9 @@
 /.idea
 /.code
 
+### VSCode ###
```
Comment (pysalt) [inline .gitignore:9]: lets move this to another commit

Comment (feodor-ra) [reply .gitignore:9]: Done
